decidim-assemblies/app/commands/decidim/assemblies/admin/update_assembly.rb

```diff
@@ -74,7 +74,28 @@ def attributes
             participatory_scope: form.participatory_scope,
             participatory_structure: form.participatory_structure,
             meta_scope: form.meta_scope,
-            show_statistics: form.show_statistics
+            show_statistics: form.show_statistics,
+            purpose_of_action: form.purpose_of_action,
+            type_of_assembly: form.type_of_assembly,
+            type_of_assembly_other: form.type_of_assembly_other,
+            date_created: form.date_created,
```

Comment:
We already have a `created_at` field, what's the difference?

Reply:
It is the date of creation of an assembly. day of birth ?

Reply:
Ah, I get it. Thanks! Maybe `born_At`, `birth_date`, but I don't have a strong opinion on this, I'll leave it to your criteria 😄

Reply:
It seems to be the creation date of the assembly, not the DB record. Maybe rename it to `creation_date`?